libraries/rush-lib/src/logic/operations/IPCOperationRunner.ts

```diff
@@ -30,6 +30,7 @@ export interface IIPCOperationRunnerOptions {
   commandForHash: string;
   persist: boolean;
   requestRun: OperationRequestRunCallback;
+  ignoredParameterNames: ReadonlyArray<string>;
```

Comment:
These should be the expanded CLI from the ignored parameters, if any where encountered while expanding the CLI for this command.